src/components/shared/Entity/DetailsForm.tsx

```diff
@@ -36,6 +37,15 @@ interface Props {
     formStateStoreName: FormStateStoreNames;
 }
 
+const getConnectorImageDetails = (connector: ConnectorWithTagDetailQuery) => {
+    return {
+        connectorId: connector.id,
+        id: connector.connector_tags[0].id,
+        imagePath: `${connector.image_name}${connector.connector_tags[0].image_tag}`,
+        iconPath: connector.image,
+    };
+};
```

Comment:
One day was might want to make some pattern or something for mapping but right now this seems fine.